test/infrastructure/docker/main.go

```diff
@@ -104,6 +104,9 @@ func initFlags(fs *pflag.FlagSet) {
 
 func main() {
 	rand.Seed(time.Now().UnixNano())
+	if _, err := os.ReadDir("/tmp/"); err != nil {
+		panic(err)
+	}
```

Comment:
```suggestion
```
:)

Reply:
Sorry, what is the suggested change here?

Reply:
I'm guessing you meant to change the panic to `os.exit(1)` to match the other exits from main. Update coming with that, but please let me know if there is a different point I've missed. Thanks!